docs/oerpolicies.md

```diff
@@ -27,4 +27,10 @@ The project has started just recently and will be elaborated in the next weeks a
   Open in a new window
 </a>
 
-<iframe src="https://oerworldmap.org/kibana/app/kibana#/dashboard/3f24aa90-e370-11e8-bc1a-bd36147d8400?embed=true&_g=()" height="750" width="800" style="border:0; width: 100%; margin: 0 auto;"></iframe>
\ No newline at end of file
+<iframe
+  src="https://oerworldmap.org/kibana/app/kibana#/dashboard/3f24aa90-e370-11e8-bc1a-bd36147d8400?embed=true&_g=()"
+  data-scope="filter.about.additionalType.@id=https://oerworldmap.org/assets/json/publications.json#policy"
```

Comment:
Once https://github.com/hbz/oerworldmap/issues/1628 is deployed, this will have to be updated to `about.@type:Policy`.